packages/@uppy/core/src/index.test.js

```diff
@@ -1604,6 +1604,34 @@ describe('src/Core', () => {
       }
     })
 
+    it('should enforce the maxTotalFileSize rule', () => {
+      const core = new Core({
+        restrictions: {
+          maxTotalFileSize: 34000
+        }
+      })
+
+      core.addFile({
+        source: 'jest',
+        name: 'foo.jpg',
+        type: 'image/jpeg',
+        data: new File([sampleImage], { type: 'image/jpeg' })
+      })
+
+      try {
+        core.addFile({
+          source: 'jest',
+          name: 'foo1.jpg',
+          type: 'image/jpeg',
+          data: new File([sampleImage], { type: 'image/jpeg' })
+        })
+        throw new Error('should have thrown')
```

Comment:
Is there some reason https://jestjs.io/docs/en/expect#tothrowerror wouldn't work?